packages/babel-preset/src/assets/replaceAssetsWithImports.ts

```diff
@@ -29,6 +30,39 @@ export function replaceAssetsWithImports(
     return assetIdentifiers.get(assetPath) as t.Identifier;
   }
 
+  function buildTemplateLiteralFromValue(value: string): t.TemplateLiteral {
+    const tokens = tokenize(value);
+
+    const quasis: t.TemplateElement[] = [];
```

Comment:
what are quasis ?

Reply:
It's how Babel names string in template literals i.e. `foo` from `foo${bar}`:

![image](https://user-images.githubusercontent.com/14183168/195193030-0d307b16-5c7b-4e45-93a6-c287ab234947.png)

https://babeljs.io/docs/en/babel-types#templateliteral